azure-maven-plugin/README.md

```diff
@@ -0,0 +1,54 @@
+# Maven Plugin for Azure
+
+## Table of Content
+  - [Overview](#overview)
+    - [AzureMaven Plugin](#azure-login-overview)
+  - [Prerequisites](#prerequisites)
+  - [Goals](#goals)
+    - [`azure:login`](#azure-login)
+    - [`azure:select-subscription`](#azure-select-subscription)
+    - [`azure:logout`](#azure-logout)
+
+## Overview
+
+<a name="azure-login-overview"></a>
```

Comment:
Do we need this HTML tag?

Reply:
this  link is to define linke for the ### Azure Maven plugin